Builds with wasm-bindgen>=0.2.96 fail with this message:

{
  "reason": "compiler-message",
  "package_id": "registry+https://github.com/rust-lang/crates.io-index#wasm-bindgen-macro@0.2.97",
  "manifest_path": "/home/stas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-macro-0.2.97/Cargo.toml",
  "target": {
    "kind": [
      "proc-macro"
    ],
    "crate_types": [
      "proc-macro"
    ],
    "name": "wasm_bindgen_macro",
    "src_path": "/home/stas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-macro-0.2.97/src/lib.rs",
    "edition": "2021",
    "doc": true,
    "doctest": true,
    "test": true
  },
  "message": {
    "rendered": "error[E0554]: `#![feature]` may not be used on the stable release channel\n --> /home/stas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-macro-0.2.97/src/lib.rs:4:5\n  |\n4 |     feature(allow_internal_unstable),\n  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n",
    "$message_type": "diagnostic",
    "children": [],
    "code": {
      "code": "E0554",
      "explanation": "Feature attributes are only allowed on the nightly release channel. Stable or\nbeta compilers will not comply.\n\nErroneous code example:\n\n```ignore (depends on release channel)\n#![feature(lang_items)] // error: `#![feature]` may not be used on the\n                        //        stable release channel\n```\n\nIf you need the feature, make sure to use a nightly release of the compiler\n(but be warned that the feature may be removed or altered in the future).\n"
    },
    "level": "error",
    "message": "`#![feature]` may not be used on the stable release channel",
    "spans": [
      {
        "byte_end": 194,
        "byte_start": 162,
        "column_end": 37,
        "column_start": 5,
        "expansion": null,
        "file_name": "/home/stas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-macro-0.2.97/src/lib.rs",
        "is_primary": true,
        "label": null,
        "line_end": 4,
        "line_start": 4,
        "suggested_replacement": null,
        "suggestion_applicability": null,
        "text": [
          {
            "highlight_end": 37,
            "highlight_start": 5,
            "text": "    feature(allow_internal_unstable),"
          }
        ]
      }
    ]
  }
}

diff --git a/cargo-auditable/tests/fixtures/wasm_crate/Cargo.toml b/cargo-auditable/tests/fixtures/wasm_crate/Cargo.toml
index 77674de..902baa0 100644
--- a/cargo-auditable/tests/fixtures/wasm_crate/Cargo.toml
+++ b/cargo-auditable/tests/fixtures/wasm_crate/Cargo.toml
@@ -11,7 +11,7 @@ crate-type = ["cdylib", "rlib"]
 default = ["console_error_panic_hook"]
 
 [dependencies]
-wasm-bindgen = "0.2.84"
+wasm-bindgen = "=0.2.95"
 wee_alloc = "0.4.5"
 
 # The `console_error_panic_hook` crate provides better debugging of panics by
